Account/SyncAuthState.swift

```diff
@@ -57,6 +58,15 @@ open class FirefoxAccountSyncAuthState: SyncAuthState {
     public var deviceID: String? {
         return account.deviceRegistration?.id
     }
+    var _engineEnablements: [String: Bool]?
+    public var enginesEnablements: [String : Bool]? {
+        get {
```

Comment:
I believe Swift supports public getter and private setter. Might try that instead, otherwise you might as well just skip this accessor pair…